frontend/public/components/row-filter.jsx

```diff
@@ -74,7 +74,7 @@ class CheckBoxes_ extends React.Component {
   }
 
   render () {
-    const checkboxes = _.map(this.props.items, ({id, title}) => {
+    const checkboxes = _.map(_.sortBy(this.props.items, item => item.title), ({id, title}) => {
```

Comment:
Since the list never changes, let's just declare them in order when we create the array instead of sorting pragmatically. This will resort the list on every render...

I would be OK having Opaque last since it's different than the others.